CHANGELOG.md

```diff
@@ -14,6 +16,8 @@ This file is used to list changes made in each version of the haproxy cookbook.
 
 ## 12.0.1 - *2021-06-01*
 
+- Standardise files with files in sous-chefs/repo-management
```

Comment:
hm...this doesn't seem right that it duplicated the line under an existing release @Xorima

Reply:
This was missed in a previous release so I was just adding it here.